WalletWasabi.Fluent/Styles/Styles.xaml

```diff
@@ -14,6 +14,19 @@
     <Setter Property="IsPaneOpen" Value="True" />
     <Setter Property="PaneBackground" Value="{x:Null}" />
   </Style>
+  <Style Selector="Button.invisible"> 
```

Comment:
Get rid of the button... anything can be clicked.

Reply:
this style is still useful, im using this for Back buttons in dialogs